best_practices/creating-the-project.rst

```diff
@@ -110,6 +110,13 @@ Symfony documentation uses the AppBundle name.
     There is no need to prefix the AppBundle with your own vendor (e.g.
     AcmeAppBundle), because this application bundle is never going to be
     shared.
+    
+.. note::
+    
+    Another reason to create a new bundle is when you're overriding something 
+    in a vendor's bundle. For instance, if you have to override an action of 
+    the FOSUserBundle, then you should create a UserBundle that would be used 
```

Comment:
I suggest using a different example than FOSUserBundle. Once I finally release its 2.0 version, people would generally not need to overwrite our actions, because we provide lots of extension points in them. And overwriting FOSUserBundle 2 actions is the worse way to change the behavior.